components/DateInput.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Controller, Control } from 'react-hook-form';
+import { LocalizationProvider, DatePicker } from '@mui/x-date-pickers';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+
+type DateInputProps = {
+  name: string;
+  control: Control;
+  label?: string;
+  rules?: object;
+  defaultValue?: string;
+};
+
+const DateInput = ({
+  name,
+  control,
+  label,
+  rules = {},
+  defaultValue = '',
```

Comment:
If you set `defaultValue` like this, then default value for all date input fields in the form will be empty string. We don't want that.